app/src/main/res/values/environment_config.xml

```diff
@@ -15,25 +15,8 @@
     <string name="torusBaseUrl">https://metadata.tor.us</string>
     <string name="torusTestUrl">http://beardict.net:5051</string>
 
-    <string name="torusDebugVerifier">testTkeyD1</string>
-    <string name="torusFeatureVerifier">testTketF2</string>
-    <string name="torusReleaseVerifier">key-app-google-cyan</string>
-
-    <!--    DECOMMENT THE LINE NEEDED DEPENDING ON BUILD-->
-    <!--    FOR LOCAL RELEASE BUILDS-->
-    <string name="torusReleaseSubVerifier">android</string>
-
-    <!--    FOR PLAY STORE RELEASE BUILDS-->
-    <!--    <string name="torusReleaseSubVerifier">android-1</string>-->
```

Comment:
please check again for correctness
in our release branch the subverifiers are totally wrong 
torusReleaseSubVerifier should be android-1, not android-2